pkg/ebpftracer/tracer_decode_test.go

```diff
@@ -199,34 +287,35 @@ func newMockContainersClient() *MockContainerClient {
 	}
 }
 
-func buildTestEventData(t *testing.T) []byte {
-	ctx := TracerEventContextT{
-		Ts: 11,
-		Task: tracerTaskContextT{
-			CgroupId:    22,
-			Pid:         543,
-			Tid:         77,
-			Ppid:        4567,
-			HostPid:     5430,
-			HostTid:     124,
-			HostPpid:    555,
-			NodeHostPid: 21345,
-			Uid:         98123,
-			MntId:       1357,
-			PidId:       3758,
-			Comm:        [16]int8{0x48},
-		},
-		Eventid: uint32(events.ProcessOomKilled),
+func testTaskContext() tracerTaskContextT {
+	return tracerTaskContextT{
+		CgroupId:    22,
+		Pid:         543,
+		Tid:         77,
+		Ppid:        4567,
+		HostPid:     5430,
+		HostTid:     124,
+		HostPpid:    555,
+		NodeHostPid: 21345,
+		Uid:         98123,
+		MntId:       1357,
+		PidId:       3758,
+		Comm:        [16]int8{0x48},
 	}
+}
 
-	dataBuf := &bytes.Buffer{}
-
-	err := binary.Write(dataBuf, binary.LittleEndian, ctx)
-	require.NoError(t, err)
-
-	// writes argument length
-	err = binary.Write(dataBuf, binary.LittleEndian, uint8(0))
-	require.NoError(t, err)
+func schedProcessExecTestEvent() *TracerEventContextT {
```

Comment:
Those test event functions are only used in one test, would be nice to use them directly as in

```go
TracerEventContextT{
 		Ts:      10,
 		Eventid: uint32(events.SchedProcessExec),
 		Task:    testTaskContext(),
 	}.Encode()
```